src/containerbuddy/config.go

```diff
@@ -95,6 +102,10 @@ func loadConfig() (*Config, error) {
 		return nil, errors.New("More than one discovery backend defined")
 	}
 
+	if config.StopTimeout == 0 {
+		config.StopTimeout = 10
```

Comment:
I think maybe I should document this a bit better. My intention here was to allow someone to set stopTimeout = -1 and get SIGKILL semantics on the managed application (while still allowing Containerbuddy to clean up)